reference_data/build_package_data.R

```diff
@@ -0,0 +1,11 @@
+cleaning_config <- yaml::read_yaml(here::here("reference_data","data_cleaning.yaml"))
```

Comment:
spaces after ","

Reply:
This is actually a symptom of a slightly larger issue that I would address seperately; #103